src/pages/Preprocess.js

```diff
@@ -159,13 +159,13 @@ class App extends Component {
             <h5 className="card-title">Dataset {this.props.match.params.user}</h5>
             <ul className="nav nav-tabs dataset-tabs">
               <li className="nav-item" onClick={this.handleClickTab('vis')}>
-                <a className={`nav-link ${this.state.currentTab === 'vis' && 'active'}`} href="#">Vis</a>
+                <button type="button" className={`btn btn-light ${this.state.currentTab === 'vis' && 'active'}`}>Vis</button>
```

Comment:
Replacing `<a>` with `<button>` actually messes the CSS due to the nature of Bootstrap. We can fix this by editing the CSS ourselves or using React-Bootstrap